rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2019-09-01
```

Comment:
Not necessary, I've just been pinning the toolchain on these PRs because I develop on Windows sometimes and the build has been broken with current nightly. I'll remove it before readying for review.